src/behaviors.js

```diff
@@ -47,7 +47,7 @@ Marionette.Behaviors = (function(Marionette, _) {
         behaviorEvents = Marionette.normalizeUIKeys(behaviorEvents, getBehaviorsUI(b));
 
         var j = 0;
-        _.each(behaviorEvents, function(behaviour, key) {
+        _.each(behaviorEvents, function(behaviorEvent, key) {
```

Comment:
It's a item from the behaviorEvents collection, so behaviorEvent seems to be the correct phrase. Handler could also be working, but I find it a bit more abstract.